test/blackbox-tests/test-cases/melange/private.t/run.t

```diff
@@ -12,13 +12,15 @@ Cmj rules should not include --bs-package-name
   0
   [1]
 
+  $ output=inside/output
+
 Js rules should include module type
-  $ dune rules inside/output/app/app__B.js | 
+  $ dune rules $output/inside/app/app__B.js | 
   > grep -e "--bs-module-type" --after-context=1 
       --bs-module-type
       es6
 
 Build js files
-  $ dune build inside/output/melange__C.js
-  $ node _build/default/inside/output/melange__C.js
+  $ dune build $output/inside/melange__C.js
```

Comment:
I wonder if we could mitigate the ugliness of the new paths like `inside/output/inside/...` by placing the js artifacts resulting from the `melange.emit` entries at the top level of the emit target folder. For example, in this case, the path to js file would be `inside/output/melange__C.js` (the inner `/inside` subpath is gone).

The downsides are that there might be naming "collisions" with js artifacts that might be produced from that top level folder. It might also be more confusing for users as it is not consistent behavior.

cc @anmonteiro

Reply:
I don't think that makes sense. I believe the JS files end up where they are in the source tree, just nested under `inside/output` because that's where the `emit` stanza lives.

The `emit` stanza is still in the `inside` folder, so it makes sense for the JS files to live there.

If you want these to be moved to the top level you can either:
- move the `emit` stanza somewhere else
- add a directory target rule that copies the emit stanza output dir to the top leve.